### DISCUSSION TITLE
NVIDIA drivers suddenly gone and computer doesn't recognize 4060 gpu. NVIDIA control panel: "not available because screen is not connected to nvidia gpu"

### DISCUSSION THREAD

**Discussion by martinboson** [Atlas-OS/Atlas#1233] [Help & Support]
I just started the computer last night and it boots into 1024x768 resolution and drivers are just gone.
I reinstalled drivers right away, then today when I woke up they were gone again. tech forum reddit.com/techsupport won't touch the issue because of altered windows installation. Is there a way to find a log and find out what the hell happened?
I used the official guide using NVCleaninstall.exe

Processor	AMD Ryzen 5 5600X 6-Core Processor                3.70 GHz
Installed RAM	32,0 GB
System type	64-bit operating system, x64-based processor
GPU                 NVIDIA ASUS RTX 4060, DRIVER VER 561.09

**Comment by Ast3risk-ops** [comment]
Reinstall your drivers (choose Custom and tick the Clean Install box during install which will remove the broken ones)

**Reply by martinboson** [reply]
>I reinstalled drivers right away

>I used the official guide using NVCleaninstall.exe

**Reply by Ast3risk-ops** [reply]
> > I reinstalled drivers right away
> 
> > I used the official guide using NVCleaninstall.exe

NVCleanInstall does not do a clean install by default. That box must be checked during setup.

You can also try removing your  graphics driver with DDU (display driver uninstaller) beforehand.

**Reply by martinboson** [reply]
that's implied, since I followed the guide from here: 

>After the driver is downloaded tick the following:

>    Disable Installer Telemetry & Advertising
>    Perform a Clean Installation
>    Disable [Multiplane Overlay (MPO)](https://docs.atlasos.net/getting-started/post-installation/drivers/amd/#disable-multi-plane-overlay-mpo)
>    Disable Driver Telemetry in Show Expert Tweaks
>    Use method compatible with Easy-Anti-Cheat
>    Automatically accept the "driver unsigned" warning

>Click Install and continue with the NVIDIA driver installation as usual.
